_codegen/main.go

```diff
@@ -1,3 +1,31 @@
+/*
+The MIT License (MIT)
+
+Copyright (c) 2015,2016 Ernesto Jiménez
+Copyright (c) 2019 Leigh McCulloch
+Copyright (c) 2020 Matt Gorzka
+Copyright (c) 2024 Simon Schulte
+Copyright (c) 2023,2025 Olivier Mengué
```

Comment:
This is unusual and would require careful attention to keep up to date. Follow the same pattern as testify and allow VCS to be the source of the names: https://github.com/stretchr/testify/blob/6c30310a8d0c34a6aa840c28898363f2d434147c/LICENSE#L3

```suggestion
Copyright (c) 2015-2025 Ernesto Jiménez and contributors
```

Reply:
I was tempted to do that, but to stay on the safe side, I chose to mention all contributors I found in the Git history.

As this is not really engineering, but legal stuff, I would be more confident if I you could give me some references to back the choice of a particular style of copyright mentions.